src/transactions/OfferExchange.h

```diff
@@ -283,6 +283,11 @@ int64_t adjustOffer(Price const& price, int64_t maxWheatSend,
 bool checkPriceErrorBound(Price price, int64_t wheatReceive, int64_t sheepSend,
                           bool canFavorWheat);
 
+bool exchangeWithPool(int64_t reservesToPool, int64_t maxSendToPool,
```

Comment:
a comment on naming convention used would be great: other places we use "wheat/sheep", so this is new